features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/WindowInspector.kt

```diff
@@ -10,54 +10,56 @@ import android.annotation.SuppressLint
 import android.os.Build
 import android.view.View
 import android.view.inspector.WindowInspector
+import com.datadog.android.api.InternalLogger
 import java.lang.reflect.Field
 
 @SuppressLint("PrivateApi")
 internal object WindowInspector {
 
-    @Suppress("SwallowedException", "TooGenericExceptionCaught")
+    internal const val FAILED_TO_RETRIEVE_DECOR_VIEWS_ERROR_MESSAGE =
+        "SR WindowInspector failed to retrieve the decor views"
+
+    @Suppress("SwallowedException", "TooGenericExceptionCaught", "UnsafeThirdPartyFunctionCall")
     private val GLOBAL_WM_CLASS by lazy {
-        try {
```

Comment:
Why remove the try catches here?

Reply:
because they are all handled in that method there `getGlobalViews`